tests/tiup-cluster/script/tikv_cdc.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+set -eu
+
+function tikv_cdc_test() {
+	mkdir -p ~/.tiup/bin/
+
+	version="nightly"
+	topo_name="tikv_cdc"
+	test_tls=false
+	mirror="https://tiup-mirrors.pingcap.com/"
+	tikv_cdc_patch=""
+
+	while [[ $# -gt 0 ]]; do
+		case $1 in
+		--version)
+			version="$2"
+			shift
+			shift
+			;;
+		--topo)
+			topo_name="$2"
+			shift
+			shift
+			;;
+		--tls)
+			test_tls=true
+			shift
+			;;
+		--staging)
+			mirror="http://staging.tiup-server.pingcap.net"
+			shift
+			;;
+		--tikv-cdc-patch)
+			tikv_cdc_patch="$2"
+			shift
+			shift
+			;;
+		esac
+	done
+
+	# set mirror
+	tiup mirror set $mirror
```

Comment:
as the staging mirror is not used, i think it's not necessary to build `tiup` and use it to manually set the mirror.

Reply:
`tiup` is also used for installing `tikv-cdc` component for test case of `tiup cluster patch`. I remove the `tiup mirror` here, but keep the `tiup` target & function.